app/components/FormFields/AvatarField.js

```diff
@@ -18,6 +26,8 @@ import WarningMessage, { Div as WarningMessageDiv } from './WarningMessage';
 
 // < 1000 chars - hash, >> 1000 - is base64 (new image)
 export const HASH_CHARS_LIMIT = 1000;
+// size in Mb
+const IMG_SIZE_LIMIT = 2;
```

Comment:
It is a good practice to mention measurement in the naming of the constant. If rename to `IMG_SIZE_LIMIT_MB` then you can tell from the name that it is in Mb and do not have to try find comment or documentation.